src/optimize.ts

```diff
@@ -329,7 +329,11 @@ async function processImage(
   /*
    * Check for external images
    */
-  if (!isLocal(attrib_src)) {
+  if (!isLocal(attrib_src) || isIncluded(
+    attrib_src,
+    config.image.cdn.src_include,
+    config.image.cdn.src_exclude
```

Comment:
Hm now realizing this isn't great as if image is not local user cannot exclude it 🤔